### PR TITLE
Make importable in another repo

### PR DESCRIPTION
**Pull request by scaubrey**
Piling onto the previous pull request. Would be good to get this in asap so we can point the simulator to the master branch here.

### CODE REVIEW

**Review by jameno — DISMISSED** [review]
LGTM!

File: tidepool_data_science_models/models/simple_metabolism_model.py [hunk 1]
```diff
@@ -91,9 +91,9 @@ def run(self, carb_amount, insulin_amount=np.nan, num_hours=8, five_min=True):
         if num_hours > 24:
             raise ValueError("Number of hours for simulation can't be more than 24.")
 
-        if carb_amount <= 0 and insulin_amount <= 0:
-            raise ValueError("Insulin or carbs must be greater than zero.")
-
+        if carb_amount < 0:  # Note: insulin can be negative
```
Comment (jameno) [inline tidepool_data_science_models/models/simple_metabolism_model.py]: Looks good!

**Review by jameno — APPROVED** [review]
A quick scan of files didn't set off any flags for me and all tests are indeed passing on my local machine as well. 

LGTM

**Review by ed-nykaza — APPROVED** [review]
LGTM, from a high-level overview. I will do a deeper dive later.

Here are some small things you will probably want to change:

- [ ] fig.show(“notebook”) needs to be added for capatability with jupyter notebook and jupyter will need to be added to the environment. I am not sure if adding "notebook" will break the colab version?
- [ ] looks like data folder and tox.ini are empty
- [ ] create conda-environment.yml file:
name: tidepool-data-science-models
channels:
  - defaults
  - conda-forge
dependencies:
  - pip==20.0.2
  - python=3.6
  - pip:
      - -r file:requirements.txt

**Pull request merged**